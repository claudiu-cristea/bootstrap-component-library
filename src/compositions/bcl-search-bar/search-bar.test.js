import { renderTwigFileAsNode } from "@openeuropa/bcl-test-utils";
import demoData from "@openeuropa/bcl-search-bar/dataSearch";

const template = "@oe-bcl/bcl-search-bar/search-bar.html.twig";
const render = (params) => renderTwigFileAsNode(template, params);

describe("OE - Search Bar", () => {
  test(`renders correctly`, () => {
    expect.assertions(1);

    return expect(render(demoData)).resolves.toMatchSnapshot();
  });
});
