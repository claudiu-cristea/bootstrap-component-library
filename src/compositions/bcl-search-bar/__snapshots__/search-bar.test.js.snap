// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OE - Search Bar renders correctly 1`] = `
<jest>
  <p
    class="form-label"
  >
    Search bar label
  </p>
  <div
    class="bcl-search-bar"
  >
    <svg
      class="bcl-search-bar__icon bi icon--xs"
    >
      <use
        xlink:href="#search"
      />
    </svg>
    <label
      class="form-label"
    >
      Search bar label
    </label>
    <input
      class="bcl-search-bar__input form-control"
      placeholder="Search"
      type="search"
    />
  </div>
</jest>
`;
